app/javascript/mastodon/features/account_timeline/index.js

```diff
@@ -63,41 +63,48 @@ class AccountTimeline extends ImmutablePureComponent {
   };
 
   componentWillMount () {
-    const { params: { accountId }, withReplies } = this.props;
+    const { params: { accountId }, withReplies, dispatch } = this.props;
 
-    this.props.dispatch(fetchAccount(accountId));
-    this.props.dispatch(fetchAccountIdentityProofs(accountId));
+    dispatch(fetchAccount(accountId));
+    dispatch(fetchAccountIdentityProofs(accountId));
 
     if (!withReplies) {
-      this.props.dispatch(expandAccountFeaturedTimeline(accountId));
+      dispatch(expandAccountFeaturedTimeline(accountId));
     }
 
-    this.props.dispatch(expandAccountTimeline(accountId, { withReplies }));
-  }
+    dispatch(expandAccountTimeline(accountId, { withReplies }));
 
-  componentDidMount () {
-    if (this.props.params.accountId === me) {
-      this.disconnect = this.props.dispatch(connectUserTimelineStream(me));
```

Comment:
It's probably better to use `componentDidMount` for all of that, `componentWillMount` is deprecated and since much of those things are API queries anyway, the difference wouldn't be noticeable.